build.sbt

```diff
@@ -92,6 +92,70 @@ lazy val shocon = crossProject.in(file(".")).
 lazy val shoconJVM = shocon.jvm
 lazy val shoconJS = shocon.js
 
+lazy val plugin = project
+  .settings(
+    commonSettings ++
+    sonatypeSettings ++
+    publishingSettings: _*)
+  .settings(
+    name := "sbt-shocon",
+    description := "sbt plugin for shocon",
+    sbtPlugin := true,
+    scalaVersion := "2.10.6",
+    crossScalaVersions := Seq("2.10.6"),
+    addSbtPlugin("org.scala-js" % "sbt-scalajs" % scalaJSVersion),
+    scalacOptions ++= Seq(
+      "-feature",
+      "-unchecked",
+      "-language:implicitConversions"),
+    sourceGenerators in Compile += Def.task {
+      val file = (sourceManaged in Compile).value / "Version.scala"
+      IO.write(file,
+        s"""package eu.unicredit.shocon.sbtplugin
+        |object Version { val shoconVersion = "${version.value}" }
+        """.stripMargin)
+      Seq(file)
+    }.taskValue,
+
+    // configuration for testing with sbt-scripted
+    ScriptedPlugin.scriptedSettings,
+    scriptedLaunchOpts <++= version apply { version =>
+      Seq("-Xmx1024M", "-Dplugin.version=" + version)
+    },
+    scriptedBufferLog := false,
+    publishLocal <<= publishLocal.dependsOn((publishLocal in (shoconJS)))
```

Comment:
this should become something like:
`publishLocal := {publishLocal.dependsOn((publishLocal in (shoconJS)).value}`